doc/api/esm.md

```diff
@@ -854,33 +784,27 @@ export function resolve(specifier, context, defaultResolve) {
   return defaultResolve(specifier, context, defaultResolve);
 }
```

Comment:
Do you mean to remove the 3rd argument?

Reply:
Just noticed it was identical to before and was wondering if it needs any changes for the new APIs.

Reply:
The 3rd argument technically is not correct, but it doesn't _actually_ matter for now (if they pass it, it'll just be ignored). It may change based on the outcome of #36954

The rest is accurate (it didn't change).